seldon-core-operator/rockcraft.yaml

```diff
@@ -0,0 +1,115 @@
+name: seldon-core-operator
+summary: Convert your ML models or language wrappers into microservices
+description: |
+  Seldon core converts your ML models (Tensorflow, Pytorch, H2o, etc.) or
+  language wrappers (Python, Java, etc.) into production REST/GRPC
+  microservices.
+
+  Seldon handles scaling to thousands of production machine learning models and
+  provides advanced machine learning capabilities out of the box including
+  Advanced Metrics, Request Logging, Explainers, Outlier Detectors, A/B Tests,
+  Canaries and more.
+
+version: v1.15.1_22.04_1 # version format: <KF-upstream-version>_<base-version>_<Charmed-KF-version>
+license: Apache-2.0
+base: ubuntu:22.04
+services:
+  manager:
+    override: replace
+    summary: "seldon-core service"
+    startup: enabled
+    user: ubuntu
+    command: "/opt/seldon-core-operator/manager --enable-leader-election --webhook-port 4443 --create-resources true"
+platforms:
+  amd64:
+
+parts:
+  seldon-core-operator:
+    plugin: go
+    source: https://github.com/SeldonIO/seldon-core
+    source-type: git
+    source-tag: v1.15.1
+    build-snaps:
+      - go
+      - kustomize/latest/stable
```

Comment:
How about specifying version for go with 
```
build-snaps:
 - go/1.17/stable
```

Reply:
If we do not specify version, latest go will be used which is v 1.20. It has less CVEs.